<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCalAreaUnderCurve</class>
 <widget class="QDialog" name="dlgCalAreaUnderCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infinity Calculator &gt;&gt; Calculate Area Under the Curve</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/resources/resources/windowicon.png</selectedon>
   </iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="txtFormula">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #dbdbdb;	
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFormula">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Formula</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMinRange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #dbdbdb;	
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMinRange">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>65</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min Range</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMaxRange">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>60</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #dbdbdb;	
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaxRange">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>65</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCalAreaUndertheCurve">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Area Under the Curve (in Units)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCalAreaUndertheCurve">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>165</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: #dbdbdb;	
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCalculate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>115</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lao UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 	background-color: #dbdbdb;
    border: 2px;
	color:#464747;

}

QPushButton:hover{background-color:#40bd9e; color:white;}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
